packages/rum-recorder/src/domain/rrweb/record.ts

```diff
@@ -15,30 +15,24 @@ export function record(options: RecordOptions = {}): RecordAPI {
 
   const mutationController = new MutationController()
 
-  const takeFullSnapshot = (isCheckout = false) => {
+  const takeFullSnapshot = () => {
```

Comment:
there are 2 remaining checkout references in record.spec:

- https://github.com/DataDog/browser-sdk/blob/a99bca93d57a4845fbe4d7808f5d70af54ede264/packages/rum-recorder/src/domain/rrweb/record.spec.ts#L43
- https://github.com/DataDog/browser-sdk/blob/a99bca93d57a4845fbe4d7808f5d70af54ede264/packages/rum-recorder/src/domain/rrweb/record.spec.ts#L55

do they still make sense?